src/components/ProjectTags.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
 
-const ProjectTags = ({ tags, isUrgent }) => (
+const ProjectTags = ({ tags, isUrgent, selectedStatus={} }) => (
   <>
-    {isUrgent && <span className="project-tag project-tag-urgent">Urgent</span>}
+    {isUrgent &&
+      <span
+        className={`project-tag project-tag-urgent ${selectedStatus['Urgent'] ? 'project-tag-urgent-selected' : ''}`}
```

Comment:
i hate looking at this lol should i bother installing cx or classnames or something